src/Storage/FilenameStrategyInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MakinaCorpus\DbToolsBundle\Storage;
+
+interface FilenameStrategyInterface
+{
+    /**
+     * Generate dump file name.
+     */
+    public function generateFilename(
+        string $storageRootDir,
+        string $connectionName = 'default',
+        string $extension = 'sql',
+        bool $anonymized = false
+    ): string;
+
+    /**
+     * If your strategy doesn't store backups in the configured root directory
+     * then return here a root path in which backups will be lookup by the
+     * storage manager when the restore command is run.
+     *
+     * Most implementation should just return "null" here.
+     */
+    public function getRootDir(): ?string;
```

Comment:
This method name is confusing but I don't know how to make it more explicit

Reply:
At least the name is consistent with the parameter name, I have better name myself.